### PR TITLE
Add class label description to R2V data products

### PR DESCRIPTION
**Pull request by bradleynull**
Added the class label descriptions to the R2V documentation so the
`class_label` can be used in the object detection and tracking output
JSON.

### CODE REVIEW

File: docs/r2v/data_processing_output.md [hunk 1]
```diff
@@ -186,6 +186,32 @@ and tracking. The JSON for the object annotations includes:
 * `bbox`: The bounding box in side of the original image
 * `mask`: THe RLE mask for the object in the image
 
+The labels for the class correlate with the following table:
+
+| Classification  |	Label | 
+| ----- | --------------------- |
```
Comment (devinaconley) [inline docs/r2v/data_processing_output.md:192]: add null label `0`

File: docs/r2v/data_processing_output.md [hunk 2]
```diff
@@ -186,6 +186,32 @@ and tracking. The JSON for the object annotations includes:
...
+| motorcycle |	4 |
+| bus |	6 |
+| truck |	8 |
+| fence/guardrail |	5 |
+| traffic light |	10 |
+| person |	11 |
+| bicycle |	12 |
+| building |	15 |
+| traffic signs |	20 |
+| lane markers |	70 |
```
Comment (devinaconley) [inline docs/r2v/data_processing_output.md:203]: we aren't currently classifying this in segmentation, should we remove for now?

File: docs/r2v/data_processing_output.md [hunk 3]
```diff
@@ -186,6 +186,32 @@ and tracking. The JSON for the object annotations includes:
...
+| terrain |	80 |
+| foliage |	85 |
+| gravel |	100 |
+| power lines |	110 |
+| pylons |	115 |
```
Comment (devinaconley) [inline docs/r2v/data_processing_output.md:208]: remove

File: docs/r2v/data_processing_output.md [hunk 4]
```diff
@@ -186,6 +186,32 @@ and tracking. The JSON for the object annotations includes:
...
+| traffic light |	10 |
+| person |	11 |
+| bicycle |	12 |
+| building |	15 |
+| traffic signs |	20 |
+| lane markers |	70 |
+| terrain |	80 |
+| foliage |	85 |
+| gravel |	100 |
+| power lines |	110 |
```
Comment (devinaconley) [inline docs/r2v/data_processing_output.md:207]: remove

File: docs/r2v/data_processing_output.md [hunk 5]
```diff
@@ -186,6 +186,32 @@ and tracking. The JSON for the object annotations includes:
...
+| fence/guardrail |	5 |
+| traffic light |	10 |
+| person |	11 |
+| bicycle |	12 |
+| building |	15 |
+| traffic signs |	20 |
+| lane markers |	70 |
+| terrain |	80 |
+| foliage |	85 |
+| gravel |	100 |
```
Comment (devinaconley) [inline docs/r2v/data_processing_output.md:206]: we aren't currently classifying this in segmentation, should we remove for now?

File: docs/r2v/data_processing_output.md [hunk 6]
```diff
@@ -186,6 +186,32 @@ and tracking. The JSON for the object annotations includes:
...
+| power lines |	110 |
+| pylons |	115 |
+| sky |	141 |
+| street light/pole |	153 |
+| road |	175 |
+| sidewalk |	190 |
+| road art |	193 |
```
Comment (devinaconley) [inline docs/r2v/data_processing_output.md:213]: remove? not sure what this one refers to

File: docs/r2v/data_processing_output.md (hunk 1 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md:192]: Added

File: docs/r2v/data_processing_output.md (hunk 2 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md]: Removed.

File: docs/r2v/data_processing_output.md (hunk 3 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md]: Removed.

File: docs/r2v/data_processing_output.md (hunk 4 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md]: Removed.

File: docs/r2v/data_processing_output.md (hunk 5 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md]: Removed.

File: docs/r2v/data_processing_output.md (hunk 6 above)
Comment (bradleynull) [reply docs/r2v/data_processing_output.md]: Removed.

**Comment by bradleynull** [comment]
@devinaconley Ready for another look!

**Pull request merged**